jscomp/syntax/src/res_comments_table.ml

```diff
@@ -1352,7 +1352,7 @@ and walk_expression expr t comments =
               {
                 txt =
                   Longident.Ldot
-                    (Longident.Ldot (Lident "Js", "Dict"), "fromArray");
+                    (Longident.Ldot (Lident "Js", "Dict"), "unsafe_create");
```

Comment:
@cknitt do we have a table where we keep these hardcoded paths?
This is only syntax level.

Reply:
I only know about jscomp/frontend/ast_literal.ml, but nothing in syntax.

Reply:
Actually why put it into Js.Dict and not a new runtime module? Then we don't need to clean that up later.

Currently we have
- `runtime/caml_*` for primitives (where we do not need to ship the `.cm*` files)
- `jscomp/other/runtime_*` (I added those recently) for runtime functions where the `.cm*` files are needed

I think it should be in one of the former, so something like `runtime/caml_dict.res`.

Reply:
Yeah I think that's a good idea, suggested here as well: https://github.com/rescript-lang/rescript-compiler/pull/6538#issuecomment-2327984286

I'll try it.